qutip/solver/multitrajresult.py

```diff
@@ -635,8 +656,7 @@ def merge(self, other, p=None):
 
         where p is a parameter between 0 and 1. Its default value is
         :math:`p_{\textrm{def}} = N / (N + N')`, N and N' being the number of
-        trajectories in the two result objects. (In the case of weighted
-        trajectories, only trajectories without absolute weights are counted.)
+        trajectories in the two result objects.
```

Comment:
Of course, there is no "right" or "wrong" for the default value of `p`, we can choose what we think is most useful.

Why I thought it useful to only count the relative trajectories here: when `p` is not specified, I wanted every randomly generated trajectory to have the same importance. In my eyes that means that they should not pick up weights. If the value is `p = p_equal`, then they do not pick up weights.

If you think that `p = self.num_trajectories / new.num_trajectories` is better, I am okay with that; I just wanted to briefly explain why I chose the other value. Your choice is maybe less surprising to users.

Reply:
If we don't count the trajectory with no jump, then there is no confusion. I believe that would be the best way to do it.